provider/util.go

```diff
@@ -55,17 +56,26 @@ func normalizeMonitor(tpl map[string]interface{}) {
 }
 
 func normalizeMonitorTriggers(triggers []interface{}) {
+	trigger_types := []string{"alerting_trigger", "anomaly_detector_trigger", "bucket_level_trigger", "document_level_trigger", "query_level_trigger"}
 	for _, t := range triggers {
```

Comment:
Just checking is there a way identify all these actions using client actions rather than hardcoding ?

Reply:
I see a new one `chained_alert_trigger`.

Reply:
Sadly I don't know of any :(